pages/posts/@id/+onBeforeRender.ts

```diff
@@ -17,6 +18,10 @@ export default async function onBeforeRender(pageContext: PageContextBuiltInServ
         // the client. Otherwise it gets executed on the server.
 
         console.log('posts/id/+onBeforeRender is fetching... id : ', id)
+
+        // Mark the query as started:
+        knownQueries.set(queryKey, /*isFetched*/ false)
```

Comment:
If this is the first page loaded, then it run on the server, and the store doesn't sync with the server, so it maybe useless.
If this page is visited through client navigation it may make sense.
But I'am not sure that is the correct syntax to update the zustand store outside of React Component.
I think it doesn't matter though. cause will check and set it in the `Page` component later.

Reply:
I will remove this, we don't need it